app/presenters/base_answers_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class BaseAnswersPresenter
+  include Rails.application.routes.url_helpers
+
+  attr_reader :tribunal_case
+
+  def initialize(tribunal_case)
+    @tribunal_case = tribunal_case
+  end
+
+  private
+
+  def row(value, options)
+    # `false` counts as blank, but is a valid value here so we can't use #blank?
+    return if value.nil? || value == ''
```

Comment:
Actually I'm not 100% sure we need to avoid `#blank?` anymore – this is from when `tribunal_case.challenged_decision` was a boolean.